src/ontology/cl-edit.owl

```diff
@@ -32547,6 +32548,15 @@ AnnotationAssertion(terms:date obo:CL_4047019 "2024-09-24T12:11:09Z"^^xsd:dateTi
 AnnotationAssertion(rdfs:label obo:CL_4047019 "early enterocyte"@en)
 EquivalentClasses(obo:CL_4047019 ObjectIntersectionOf(obo:CL_0000584 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0013740)))
 
+# Class: obo:CL_4047021 (laminin enterocyte of colon)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:14585148") Annotation(oboInOwl:hasDbXref "PMID:35101094") obo:IAO_0000115 obo:CL_4047021 "An enterocyte located in the epithelium of the large intestine. This cell expresses and interacts with specific laminin isoforms, particularly laminin-332 (composed of α3, β3, and γ2 chains) and laminin-511 (composed of α5, β1, and γ1 chains). Laminin enterocytes of the colon are attached to the basement membrane via hemidesmosomes containing α6β4 integrin, which binds to laminins. This cell exhibits differential expression of laminin chains along the crypt-surface axis")
```

Comment:
These references are not relevant - new references are needed to support the definition.